backends/rabbitmq/rabbitmq_test.go

```diff
@@ -24,6 +30,7 @@ func init() {
 }
 
 var _ = Describe("Shared", func() {
+
 	Context("connect", func() {
```

Comment:
Let's name the contexts or describe's for the func's they're testing. I think this is `Write`, no?

EDIT: Read/Write

Reply:
Fixed